.github/workflows/publish.yml

```diff
@@ -26,7 +25,7 @@ jobs:
 
       - name: Build & Test
         run: |
-          npm ci
+          npm i
```

Comment:
But shouldn't it be `ci`?

Reply:
Thought changed default version in nvm but guess that doesn't work at system level.  Either one here is fine really, when we were dealing with the node version issues this was attempt to workaround